api/turing/service/router_deployment_service.go

```diff
@@ -51,10 +51,9 @@ type deploymentService struct {
 	environmentType           string
 
 	// Router configs
-	fluentdConfig           *config.FluentdConfig
-	jaegerCollectorEndpoint string
-	sentryEnabled           bool
-	sentryDSN               string
+	sentryEnabled bool
+	sentryDSN     string
+	routerDefault *config.RouterDefaults
```

Comment:
Same comment as earlier - `routerDefaults` would be more consistent with the name of the struct.